landscaper/pkg/controlplane/blueprint/blueprint.yaml

```diff
@@ -74,12 +74,54 @@ imports:
   - name: runtimeCluster
     required: true
     targetType: landscaper.gardener.cloud/kubernetes-cluster
+    type: target
   - name: virtualGarden
     required: false
     schema:
       ref: blueprint://schema/com.github.gardener.gardener.landscaper.pkg.controlplane.apis.imports.v1alpha1.VirtualGarden.yaml
   
 
+exports:
+  
+  - name: gardenerAPIServerCA
+    required: true
+    schema:
+      ref: blueprint://schema/com.github.gardener.gardener.landscaper.pkg.controlplane.apis.exports.Certificate.yaml
+  - name: gardenerAPIServerEncryptionConfiguration
+    required: true
+    schema:
+      description: GardenerAPIServerEncryptionConfiguration is the encryption configuration
+        of the Gardener API Server
+      type: string
+  - name: gardenerAPIServerTLSServing
+    required: true
+    schema:
+      ref: blueprint://schema/com.github.gardener.gardener.landscaper.pkg.controlplane.apis.exports.Certificate.yaml
```

Comment:
Not sure if this is relevant but what if some import and export types are equal. This would result in two similar schemata?

Reply:
Yes, that is correct. We could potentially minimize the size of the blueprint by a few bytes, but do not think that it is worth it at this point (I do not know of a case where an import equals an exported field - this is also not the case here).

Reply:
that's ok for me :-)